src/python/s2_common.i

```diff
@@ -19,10 +25,36 @@
 #include "s2/s2pointutil.h"
 #include "s2/s2polygon.h"
 #include "s2/s2polyline.h"
+#include "s2/s2predicates.h"
 #include "s2/s2region_coverer.h"
 #include "s2/s2region_term_indexer.h"
 #include "s2/s2cell.h"
 #include "s2/s2cell_union.h"
+#include "s2/s2shape_index.h"
+#include "s2/mutable_s2shape_index.h"
+
+class S2BufferOperationOptions {
```

Comment:
I assume this is a work-around for nested classes. Add comments for why this stuff is here.

Reply:
Yes - see the commit message. I'll add a similar comment to S2PolygonLayerOptions too.